docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -115,4 +113,4 @@ This limit will be applied to every container of the given component.
 +
 For the `pugin` component type, CPU requests can be described in the plug-in descriptor file, typically named `meta.yaml`.
 +
-If none of them are specified, system-wide defaults are applied (see description of `pass:[CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__REQUEST__CORES]` system property).
+If none of them are specified, system-wide defaults are applied (see description of `pass:[WORKSPACE_SIDECAR_DEFAULT__CPU__REQUEST__CORES]` system property).
```

Comment:
I am not sure if this system property exists and how plugin behaves when no default is set in this case. @davidfestal @l0rd  We are trying to make the devfile spec Che independent. Do you have any suggestions on how to approach this?